test/extended/networking/egress_router_cni.go

```diff
@@ -0,0 +1,96 @@
+package networking
+
+import (
+	"context"
+	"fmt"
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/origin/test/extended/util"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	e2edeployment "k8s.io/kubernetes/test/e2e/framework/deployment"
+	"strings"
+	"time"
+)
+
+const (
+	EgressRouterCNIE2E        = "egress-router-cni-e2e"
+	EgressRouterCNIDeployment = "egress-router-cni-deployment"
+	EgressRouterCNINadObject  = "egress-router-cni-nad"
+	// Manifests at testdata/egress-router-cni
+	EgressRouterCNIV4Manifest = "egress-router-cni-v4-cr.yaml"
+	EgressRouterCNIV6Manifest = "egress-router-cni-v6-cr.yaml"
+	EgressRouterCNILogs       = "/tmp/egress-router-log"
+	// Match patterns are based on what is in testdata/egress-router-cni manifests
+	IPv4MatchPattern = "IP Destinations: [80 UDP 10.100.3.0 8080 SCTP 203.0.113.26 80 8443 TCP 203.0.113.27 443]"
+	IPv6MatchPattern = "IP Destinations: [80 UDP 10:100:3::0 8080 SCTP 203:0:113::26 80 8443 TCP 203:0:113::27 443]"
+	TimeOut          = 1 * time.Minute
+	Interval         = 5 * time.Second
+)
+
+var _ = g.Describe("[sig-network][Feature:EgressRouterCNI]", func() {
```

Comment:
Only the Describe test description should start with uppercase char. Everything else in 'It' and 'By' should start with lower case. ExpectNoError also. WDYT?

Reply:
Yeah, I have seen the mixed use too. 
When it prints out the message to the user, it starts with the Describe message and then may append it with 'It' and 'by', so it makes sense for them not to be capitalised. 
For [Ginkgo repo examples](https://github.com/onsi/ginkgo#ginkgo-20-is-now-generally-available), I see they always have only Describe string starting with a capitalised char also.
Anyway, its just a nit.